week4.ipynb

```diff
@@ -0,0 +1,402 @@
+{
```

Comment:
<p>I would suggest a sentence or two that kind of wraps up the lesson. It helps students feel like they synthesized the information they learned here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-vaga3461/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>